### PR TITLE
Readme branch

### PR DESCRIPTION
**Pull request by mahadd29**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,100 +1,126 @@
...
+
+GitAtom must be configured using `config.yaml` prior to initialization. 
 
-### Configuration
-Edit the configuration file: `config.yaml` before initialization.  
 #### Fields:  
-| Fields | Description|
+| Field | Description|
 | --- | --- |
 | feed_id | A metadata field in Atom, not currently used other then to fill the xml.|
```
Comment (kburton501) [inline README.md]: Websites web address or unique permanent URI

File: README.md [hunk 2]
```diff
@@ -103,28 +129,22 @@ commands: [add, commit, push]
...
 
-config.yaml is created populated with default values when init is called.
-
 Site index and archive are created at initialization, and they are updated when new entries are added.
 
-### Troubleshooting
+## Troubleshooting
 Permission denied when ssh into remote server:
 
 This issue comes up when a user has multiple ssh keys to choose from.  To solve, you need to create an alias that indicates use of a specific key.  This is done by updating the config file in the remote repository.
```
Comment (kburton501) [inline README.md]: the .ssh/config is on the local system.

File: README.md (hunk 1 above)
Comment (mahadd29) [reply README.md]: updated

File: README.md (hunk 2 above)
Comment (mahadd29) [reply README.md]: updated

**Pull request merged**